_data/food.yml

```diff
@@ -18,6 +18,19 @@ websites:
       exceptions:
         text: "Bitcoin Cash is a manual payment at this time. Contact Bees Brothers: beesbrothers@gmail.com"
 
+    - name: Crypto Mercado
+      url: https://www.cryptomercado.com/
+      img: cryptomercado.png
+      twitter: cryptomercado1
+      facebook: cryptomercadodotcom
+      region: na
+      country: US
+      city: capitalize_city_name
+      bch: yes
+      btc: yes
+      othercrypto: yes
+      doc: 
```

Comment:
Add doc link or remove this empty field.

Reply:
tags can be empty, not a huge deal